test/v2/client/test_client.go

```diff
@@ -428,7 +435,7 @@ func NewTestClient(
 					VaultMonitorInterval:               time.Second * 30,
 					PutTries:                           3,
 					MaxBlobSizeBytes:                   16 * units.MiB,
-					EigenDACertVerifierOrRouterAddress: config.EigenDACertVerifierAddressQuorums0_1,
+					EigenDACertVerifierOrRouterAddress: routerAddress.Hex(),
```

Comment:
Do you know if the router outputs the cert verifier that's being used? Would be nice to be able to check in the logs that we're using the new version.